spce.html

```diff
@@ -23,9 +23,7 @@ <h1>Spacebook - the trusted way to experience short lets and serviced apartments
                 </div>
                 <div class="right-side">
                     <form action="">
-                        <div class="area">
-                            <input type="text" placeholder="Area e.g (yaba,lagos)"><br>
-                        <div>
```

Comment:
Why did you remove this?

Reply:
Good point for the inline comment tutorial. It's just basic refactoring.